apps/services/user-notification/docker-compose.base.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.3'
```

Comment:
Take a look at how other services are configured regarding docker-compose.
There is only a single docker-compose.yml and no `base`, `ci`, `dev`